.gitignore

```diff
@@ -13,6 +13,9 @@ lerna-debug.log*
 # Diagnostic reports (https://nodejs.org/api/report.html)
 report.[0-9]*.[0-9]*.[0-9]*.[0-9]*.json
 
+# MacOS
```

Comment:
It's a better practice to use [global gitignore](https://gist.github.com/subfuzion/db7f57fff2fb6998a16c) to ignore user-specific stuff 😉. It's like IDE settings, we eventually use the contributing document to suggest configs.